R/tm_g_distribution.R

```diff
@@ -260,7 +260,6 @@ ui_distribution <- function(id, ...) {
     encoding = tags$div(
       ### Reporter
       teal.reporter::add_card_button_ui(ns("add_reporter"), label = "Add Report Card"),
-      tags$br(), tags$br(),
```

Comment:
Is this enough space? It seems a bit crammed

<img width="580" height="234" alt="image" src="https://github.com/user-attachments/assets/8030eff2-40ab-4d08-9307-ca7f6776db61" />

Old

<img width="580" height="237" alt="image" src="https://github.com/user-attachments/assets/8ad1a1d8-bdbd-42b7-b285-db3cbcb4aafb" />

Reply:
I'll revert this change. I think we still need the double line breaks. In `teal.reporter` I made a change to make this a `button` instead of `a` link which broke them in next line by default so I though just a break would be enough, but looks like having this space is better indeed.